src/views/Verify/index.view.tsx

```diff
@@ -1,21 +1,31 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 import "./index.scss"
 import { useAuth0 } from "@auth0/auth0-react"
-import { resendVerificationEmail } from "../../utils/Api"
+import ResendVerification from "../../components/ResendVerification/ResendVerification"
 
 const Verify: React.FC = () => {
-  const { user } = useAuth0()
+  const { user, getAccessTokenSilently } = useAuth0()
+  const [token, setToken] = useState<string>("")
+  const [, setError] = useState<string>("")
+  useEffect(() => {
+    if (user?.email_verified) {
```

Comment:
I think we still need to pass the access token if the user is not verified right? Otherwise they won't be authorized to request a resend.